npm install error v3.0.3

Hi,

when I install version 3.0.3

I get the error

npm ERR! path /app/node_modules/styled-bootstrap-grid
       npm ERR! code EISGIT
       npm ERR! git /app/node_modules/styled-bootstrap-grid: Appears to be a git repo or submodule.
       npm ERR! git     /app/node_modules/styled-bootstrap-grid
       npm ERR! git Refusing to remove it. Update manually,
       npm ERR! git or move it out of the way first.

There is .git folder  in node-modules which causes the problem.
Can you remove it please

--- a/.npmignore
+++ b/.npmignore
@@ -1,2 +1,0 @@
-tsconfig.json
-src

--- a/package.json
+++ b/package.json
@@ -4,6 +4,9 @@
   "description": "bootstrap grid system using styled components",
   "main": "./dist/index.js",
   "types": "./dist/index.d.ts",
+  "files": [
+    "dist/"
+  ],
   "scripts": {
     "type-check": "tsc --noEmit",
     "type-check:watch": "npm run type-check -- --watch",

